.vsts-ci.yml

```
@@ -119,6 +119,9 @@ jobs:
   strategy:
     maxParallel: 3
     matrix:
+      rpkg:
```

For the consistency with Linux and macOS builds.
```suggestion
      r_package:
```